drnim/drnim.nim

```diff
@@ -1210,19 +1210,21 @@ proc mainCommand(graph: ModuleGraph) =
       when declared(system.getMaxMem): formatSize(getMaxMem()) & " peakmem"
       else: formatSize(getTotalMem()) & " totmem"
     let loc = $conf.linesCompiled
-    let build = if isDefined(conf, "danger"): "Dangerous Release"
-                elif isDefined(conf, "release"): "Release"
-                else: "Debug"
+    let build = if isDefined(conf, "danger"): "Dangerous Release build"
+                elif isDefined(conf, "release"): "Release build"
+                else: "***SLOW, DEBUG BUILD***; -d:release makes code run faster."
     let sec = formatFloat(epochTime() - conf.lastCmdTime, ffDecimal, 3)
     let project = if conf.filenameOption == foAbs: $conf.projectFull else: $conf.projectName
     rawMessage(conf, hintSuccessX, [
       "loc", loc,
       "sec", sec,
       "mem", mem,
-      "build", build,
       "project", project,
       "output", ""
       ])
+    rawMessage(conf, hintBuildMode, [
+      "build", build,
+      ])
```

Comment:
```suggestion
    rawMessage(conf, hintBuildMode, build)
```
(since you've changed to `hintBuildMode: "$1",`)

Reply:
Done. Good catch.  I agree the duplication is suboptimal, but DrNim is also pretty recent/in progress.

Reply:
=> https://github.com/nim-lang/Nim/pull/18252